operator/avs_manager.go

```diff
@@ -285,6 +288,21 @@ func (avsManager *AvsManager) RegisterOperatorWithAvs(
 	return nil
 }
 
+func (avsManager *AvsManager) DeregisterOperator(blsKeyPair *bls.KeyPair) error {
+	// TODO: 'QuorumNums' is hardcoded for now
+	quorumNumbers := eigentypes.QuorumNums{0}
+	pubKey := eigenutils.ConvertToBN254G1Point(blsKeyPair.GetPubKeyG1())
+
+	_, err := avsManager.avsWriter.DeregisterOperator(context.Background(), quorumNumbers, pubKey)
+	if err != nil {
+		avsManager.logger.Errorf("Unable to deregister operator with avs registry coordinator: %v", err)
+		return err
+	}
+	avsManager.logger.Infof("Deregistered operator with avs registry coordinator.")
```

Comment:
Could be a `Error` (using a tag for the error, like `logger.Error("msg", "err", err)`, which is how we do it in most places) and an `Info` - I see this happens in other places in `AvsManager`, not specific from this PR.